server.go

```diff
@@ -420,6 +420,23 @@ func (s *ServerImpl) SetActiveById(id string) error {
 	return fmt.Errorf("No deploy %s, run 'list' to see valid deploys", id)
 }
 
+func (s *ServerImpl) GetFullDeployIdFromShortName(deployShortName string) (string, error) {
```

Comment:
For sanity, it might make sense to require the string to be at least say 3 chars long, to avoid random things happening with typos